tests/api_connexion/endpoints/test_dag_run_endpoint.py

```diff
@@ -1046,6 +1094,10 @@ def test_should_response_200_for_matching_execution_date_logical_date(self):
             "external_trigger": True,
             "start_date": None,
             "state": "queued",
+            "data_interval_end": "2020-11-10T08:25:56.939143+00:00",
+            "data_interval_start": "2020-11-10T08:25:56.939143+00:00",
```

Comment:
Should we use this opportunity to refactor these hardcoded dates? We could have `logical_date` and `execution_date` as constants and pass them instead of these dates?

Reply:
Thanks @ephraimbuddy, made this one and another test to use constants.